cli/src/commands/start.rs

```diff
@@ -1039,4 +1046,60 @@ mod tests {
             panic!("Unexpected result of clap parsing!");
         }
     }
+
+    #[test]
+    fn parse_peers_when_ips() {
+        let arg_vec = vec!["snarkos", "start", "--peers", "127.0.0.1:3030,127.0.0.2:3030"];
+        let cli = CLI::parse_from(arg_vec);
+
+        if let Command::Start(start) = cli.command {
+            let peers = start.parse_trusted_peers();
+            assert!(peers.is_ok());
+            assert!(peers.unwrap().len() == 2, "Expected two peers");
```

Comment:
nit: `assert_eq` could be used here

Reply:
Thanks :sun_with_face: Fixed :heavy_check_mark: